highfidelity/hifi
# Toggle going to 404 redirect domain (RC73)

- Fix for MS#[18555](https://highfidelity.manuscript.com/f/cases/18555)

This PR defaults the behavior of getting redirected to the 404 domain to be disabled.  Enabling it would be by opening the `Developer > Console` after enabling the developer menu via `Settings > Developer Menu`, and setting the mode through `Settings.setValue("enableInterstitialMode", true);`

## TEST PLAN
- Launch Interface
- Interstitial mode should be disabled by default
- Try navigating to a domain that would warrant an error (i.e. via a timeout, with an unauthorized account, or protocol mismatch)
- Dialog should display with some text related to the error
- Enable `Settings > Developer Menu`
- Open `Developer > Console`
- Enter in `Settings.setValue("enableInterstitialMode", true);`
- Try navigating to a domain that would warrant an error (i.e. via a timeout, with an unauthorized account, or protocol mismatch)
- Arrive in the redirected serverless domain

Test case:
https://highfidelity.testrail.net/index.php?/cases/view/27807
